index.html

```diff
@@ -42,6 +40,185 @@ <h2 class="intro secondary-color">I am a full stack developer.</h2>
+                        <div class="language">Html</div>
+                    </div>
+                    <button class="see-project font">
+                        See Project
+                    </button>
+                </div>
+            </div>
+        </div>
+        <header class="title">About Me</header>
+        <p> I can help you build a product, feature or website.</p>
+        <p> Look through some of my work and experience!</p>
+        <p>If you like what you see and have a project you need coded,
+            don’t hestiate to contact me.</p>
+        <div class="center">
+            <button class="see-project font">
+                Get My Resume
+            </button>
+        </div>
+        <div class="center font">
+            <div class="flash-card">
+                <div class="logo-title">
+                    <div class="logo-container">
+                        <img src="./assets/languages.svg" alt="angellist">
+                    </div>
+                    <h2>Languages</h2>
+                </div>
+                <div class="items center">
+                    <div class="item secondary-color">Ruby on rails</div>
+                    <div class="item secondary-color">CSS</div>
+                    <div class="item secondary-color">Javascript</div>
+                    <div class="item secondary-color">Html</div>
+                </div>
+            </div>
+        </div>
+        <div class="center font">
+            <div class="flash-card">
+                <div class="logo-title">
+                    <div class="logo-container">
+                        <img src="./assets/frameworks.svg" alt="angellist">
+                    </div>
+                    <h2>Frameworks</h2>
+                </div>
+                <div class="items center">
+                    <div class="item secondary-color">Bootstrap</div>
+                    <div class="item secondary-color">Ruby on rails</div>
+                    <div class="item secondary-color">RSpec</div>
+                    <div class="item secondary-color">Capybara</div>
+                    <div class="item secondary-color">Selenium</div>
+                </div>
+            </div>
+        </div>
+        <div class="center font">
+            <div class="flash-card">
+                <div class="logo-title">
+                    <div class="logo-container">
+                        <img src="./assets/skills.svg" alt="angellist">
+                    </div>
+                    <h2>Skills</h2>
+                </div>
+                <div class="items center">
+                    <div class="item secondary-color">Codekit</div>
+                    <div class="item secondary-color">GitHub</div>
+                    <div class="item secondary-color">Codepen</div>
+                    <div class="item secondary-color">GitLab</div>
+                    <div class="item secondary-color">Terminal</div>
+                </div>
+            </div>
+        </div>
```

Comment:
Kindly apply `<section>` tag to the `About me` section as well.

Reply:
Also, use an `<ul><li>` instead of `<div class="items center">`. This is a project requirement